spark3/src/main/java/org/apache/iceberg/spark/actions/BaseSnapshotTableSparkAction.java

```diff
@@ -109,7 +109,8 @@ public SnapshotTable tableProperty(String property, String value) {
 
   @Override
   public SnapshotTable.Result execute() {
-    JobGroupInfo info = newJobGroupInfo("SNAPSHOT-TABLE", "SNAPSHOT-TABLE");
+    JobGroupInfo info = newJobGroupInfo("SNAPSHOT-TABLE",
+        String.format("Snapshotting table %s as %s", sourceTableIdent().toString(), destTableIdent.toString()));
```

Comment:
Do we need to call toString explicitly? Can we also put the description into a separate var so that this fits on one line?

```
String desc = String.format("Snapshotting table %s as %s", sourceTableIdent(), destTableIdent);
```